semantic/monotype.go

```diff
@@ -117,20 +125,133 @@ var (
 )
 
 func init() {
-	builder := flatbuffers.NewBuilder(1024)
-	fbsemantic.BasicStart(builder)
-	fbsemantic.BasicAddT(builder, fbsemantic.TypeBool)
-	basicBool := fbsemantic.BasicEnd(builder)
 
-	// TODO (algow): initial all basic values
+	{
+		builder := flatbuffers.NewBuilder(0)
```

Comment:
It seems like the below code can be refactored into a single function that returns the monotype. There's only one parameter that changes and that's the type.